designer/client/styles/index.scss

```diff
@@ -24,6 +24,7 @@ body {
 main {
   flex: 1;
   display: flex;
+  padding-top: 15px;
```

Comment:
why do we need this?

Reply:
![image](https://user-images.githubusercontent.com/97108643/155509751-d7bce845-d9d5-4424-bd05-1b0492d35c49.png)
Back button looks like this without it, going off the top of the page